### ISSUE TITLE
Error detected while processing function

### ISSUE THREAD

**Issue by ktreese** [roxma/vim-paste-easy#9]
I'm getting the following error on MacOS 10.12.6

I use pathogen to install plugins, and installed by:

```
➜  ~ cd ~/.vim/bundle 
➜  bundle git clone https://github.com/roxma/vim-paste-easy.git
```

```
➜  vim --version
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Apr  4 2017 18:14:54)
Included patches: 1-898, 8056
Compiled by root@apple.com
Normal version without GUI.  Features included (+) or not (-):
+acl             -farsi           -mouse_netterm   +syntax
-arabic          +file_in_path    -mouse_sgr       +tag_binary
+autocmd         +find_in_path    -mouse_sysmouse  +tag_old_static
-balloon_eval    +float           -mouse_urxvt     -tag_any_white
-browse          +folding         +mouse_xterm     -tcl
+builtin_terms   -footer          +multi_byte      +terminfo
+byte_offset     +fork()          +multi_lang      +termresponse
+cindent         -gettext         -mzscheme        +textobjects
-clientserver    -hangul_input    +netbeans_intg   +title
-clipboard       +iconv           +path_extra      -toolbar
+cmdline_compl   +insert_expand   -perl            +user_commands
+cmdline_hist    +jumplist        +persistent_undo +vertsplit
+cmdline_info    -keymap          +postscript      +virtualedit
+comments        -langmap         +printer         +visual
-conceal         +libcall         -profile         +visualextra
+cryptv          +linebreak       +python/dyn      +viminfo
+cscope          +lispindent      -python3         +vreplace
+cursorbind      +listcmds        +quickfix        +wildignore
+cursorshape     +localmap        +reltime         +wildmenu
+dialog_con      -lua             -rightleft       +windows
+diff            +menu            +ruby/dyn        +writebackup
+digraphs        +mksession       +scrollbind      -X11
-dnd             +modify_fname    +signs           -xfontset
-ebcdic          +mouse           +smartindent     -xim
-emacs_tags      -mouseshape      -sniff           -xsmp
+eval            -mouse_dec       +startuptime     -xterm_clipboard
+ex_extra        -mouse_gpm       +statusline      -xterm_save
+extra_search    -mouse_jsbterm   -sun_workshop    -xpm
   system vimrc file: "$VIM/vimrc"
     user vimrc file: "$HOME/.vimrc"
 2nd user vimrc file: "~/.vim/vimrc"
      user exrc file: "$HOME/.exrc"
  fall-back for $VIM: "/usr/share/vim"
Compilation: gcc -c -I. -Iproto -DHAVE_CONFIG_H     -g -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1       
Linking: gcc   -L/usr/local/lib -o vim        -lm -lncurses  -liconv
```

```
Error detected while processing function <SNR>13_char_inserted:
line   10:
E117: Unknown function: reltimefloat
Error detected while processing function <SNR>13_char_inserted:
line   10:
E15: Invalid expression: reltimefloat(reltime(s:start))
Error detected while processing function <SNR>13_char_inserted:
line   12:
E121: Undefined variable: l:passed
Error detected while processing function <SNR>13_char_inserted:
line   12:
E15: Invalid expression: l:passed <= 0.01
Error detected while processing function <SNR>13_char_inserted:
line   10:
E117: Unknown function: reltimefloat
Error detected while processing function <SNR>13_char_inserted:
line   10:
E15: Invalid expression: reltimefloat(reltime(s:start))
Error detected while processing function <SNR>13_char_inserted:
line   12:
E121: Undefined variable: l:passed
Error detected while processing function <SNR>13_char_inserted:
line   12:
E15: Invalid expression: l:passed <= 0.01
Error detected while processing function <SNR>13_char_inserted:
line   10:
E117: Unknown function: reltimefloat
Error detected while processing function <SNR>13_char_inserted:
line   10:
E15: Invalid expression: reltimefloat(reltime(s:start))
Error detected while processing function <SNR>13_char_inserted:
line   12:
E121: Undefined variable: l:passed
Error detected while processing function <SNR>13_char_inserted:
line   12:
E15: Invalid expression: l:passed <= 0.01
Error detected while processing function <SNR>13_char_inserted:
line   10:
E117: Unknown function: reltimefloat
Error detected while processing function <SNR>13_char_inserted:
line   10:
E15: Invalid expression: reltimefloat(reltime(s:start))
Error detected while processing function <SNR>13_char_inserted:
line   12:
E121: Undefined variable: l:passed
Error detected while processing function <SNR>13_char_inserted:
line   12:
E15: Invalid expression: l:passed <= 0.01
Error detected while processing function <SNR>13_char_inserted:
line   10:
E117: Unknown function: reltimefloat
Error detected while processing function <SNR>13_char_inserted:
```

**Comment by roxma** [comment]
The version of your vim version doesn't have `reltimefloat`

You could try a newer version.

**Comment by ktreese** [comment]
Thank you -- that's too bad.  Unfortunately, I'm not willing to deviate from the native vim installed on my Mac.

I used to use [vim-unimpaired](https://github.com/tpope/vim-unimpaired), the `yo` and `yO` would `set paste` while only in insert mode, and `set nopaste` once insert mode was exited.  I liked it, but yours was more appealing since I could stick with regular insert commands, but oh well!

I found this instead, and it's working great for MacOS X [vim-bracketed-paste](https://github.com/ConradIrwin/vim-bracketed-paste)
